### PR TITLE
clarify defer behavior

### PR DESCRIPTION
**Pull request by mirnawong1**
Sophia [raised](https://dbt-labs.slack.com/archives/C017GDLAF7D/p1699458492603959) a great question about defer's behavior, which users were reaching out to. 

adding this info to the docs so it's clear

### CODE REVIEW

**Comment by sophiad96** [comment]
Looks good to me!

**Review by sophiad96 — APPROVED** [review]

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 1]
```diff
@@ -7,9 +7,20 @@ pagination_next: "docs/cloud/cloud-cli-installation"
 ---
 
 
-[Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build all the models that come before them (upstream parents). This is powered by using a production manifest for comparison, and dbt will resolve the `{{ ref() }}` function with upstream production artifacts.
+[Defer](/reference/node-selection/defer) is a powerful feature that allows developers to only build and run and test models they've edited without having to first run and build all the models that come before them (upstream parents). This is powered by using a production manifest for comparison, and dbt will resolve the `{{ ref() }}` function with upstream production artifacts. 
 
-Both the dbt Cloud IDE and the dbt Cloud CLI allow users to natively defer to production metadata directly in their development workflows, dramatically reducing development time and warehouse spend by preventing unnecessary model builds. 
+By default, dbt follows these rules:
+
+- Defers to the production environment when there's no development schema.
```
Comment (dave-connors-3) [inline website/docs/docs/cloud/about-cloud-develop-defer.md:14]:
```suggestion
test
```

Comment (dave-connors-3) [inline website/docs/docs/cloud/about-cloud-develop-defer.md:14]:
```suggestion
- Resolves `{{ ref() }}` functions using the production locations for the parents of selected models using metadata from the production environment.
```

File: website/docs/docs/cloud/about-cloud-develop-defer.md [hunk 2]
```diff
@@ -7,9 +7,20 @@ pagination_next: "docs/cloud/cloud-cli-installation"
...
+- If a development schema exists, dbt will prioritize those changes, which minimizes development time and avoids unnecessary model builds.
```
Comment (dave-connors-3) [inline website/docs/docs/cloud/about-cloud-develop-defer.md:15]:
```suggestion
- If a development version of a deferred model exists, dbt preferentially use that table when resolving the reference. 
- Passing the `--favor-state` flag will override this behavior, and _always_ resolve refs using the production metadata, regardless of the presence of a development relation. 
```

File: website/docs/docs/cloud/about-cloud-develop-defer.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/cloud/about-cloud-develop-defer.md:14]: - dbt uses the production locations of parent models to resolve `{{ ref() }}` functions, based on metadata from the production environment

**Pull request merged**